tests/testthat/test-use-rcpp.R

```diff
@@ -18,6 +18,35 @@ test_that("use_rcpp() creates files/dirs, edits DESCRIPTION and .gitignore", {
   expect_true(all(c("*.o", "*.so", "*.dll") %in% ignores))
 })
 
+test_that("use_rcpp_armadillo() creates Makevars files and edits DESCRIPTION", {
+  pkg <- scoped_temporary_package()
+  use_roxygen_md()
+
+  use_rcpp_armadillo()
+  expect_match(desc::desc_get("LinkingTo", pkg), "RcppArmadillo")
+
+  makevars_settings <- c(
+    "CXX_STD = CXX11",
+    "PKG_CXXFLAGS = $(SHLIB_OPENMP_CXXFLAGS)",
+    "PKG_LIBS = $(SHLIB_OPENMP_CXXFLAGS) $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)"
+  )
+
+  makevars <- readLines(proj_path("src", "Makevars"))
```

Comment:
This test is fragile because you're testing if we ever change what is in the makefile, we'll have to update all these tests. I think you'd be better of just testing that the files exist, and then creating a separate test that tests that `create_makevars()` works both when the file already exists and when it doesn't.

Reply:
I've aimed to test for the existence but not the content of either `Makevar{.win}` files in the `use_rcpp_armadillo()` test.

In addition, I've also created two unit tests for `use_makevars()` (the updated name for `create_makevars()`). The first test ensures new flags are not added and the second writes `Makevars` files if none are present.